<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Data_show</class>
 <widget class="QDialog" name="Data_show">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/tubiao/guo.png);</string>
  </property>
  <widget class="QPushButton" name="P_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tubiao/10.png);
font: 14pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt; 返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="S_name">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>29</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>数据统计：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>571</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="P_high_waste">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/tubiao/10.png);
color: rgb(255, 255, 255);
font: 18pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>高峰电量统计</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="P_low_waste">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/tubiao/10.png);
color: rgb(255, 255, 255);
font: 18pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>非高峰电量统计</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="P_total_waste">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/tubiao/10.png);
color: rgb(255, 255, 255);
font: 18pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>总电量统计</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="P_dianfei">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/tubiao/10.png);
color: rgb(255, 255, 255);
font: 18pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>总电费统计</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
